src/components/singleAccordion/SingleAccordion.unit.spec.ts

```diff
@@ -0,0 +1,123 @@
+import { mount } from "@vue/test-utils";
```

Comment:
Todo: in the other projects we're using [Vue Testing Library](https://testing-library.com/docs/vue-testing-library/intro/). I think we should use this here too and refactor the tests accordingly. Reason being the [guiding principles](https://testing-library.com/docs/guiding-principles) of Testing Library (testing the application as a user would use it, rather than relying on selectors etc., as well as consistency with the other projects.

Also see the [stylgeuide on testing](https://digitalservicebund.atlassian.net/wiki/spaces/VER/pages/884310020/Frontend+conventions+and+patterns#Use-the-recommended-order-for-element-queries-in-tests.).

Reply:
I decided to leave the test as-is, since it tests low-level features and implementation details that are harder to test with testing-library.